### DISCUSSION TITLE
Vercel Slack Integration: Re-authentication failed, just redirect to the same page

### DISCUSSION THREAD

**Discussion by dickyindra** [vercel/community#4418] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/*/integrations/icfg_SkdDuL0deGbwdcVtpp5BjRnb

### Steps to Reproduce

After setup 2fa in Slack, Vercel doesn't send notifications anymore in Slack.
I tried checking the configuration on Vercel Integration, it shows "Reauthentication required"

When I try to re-authenticate, it redirects me to the same page

https://github.com/vercel/community/assets/13186704/a072d463-6d10-4b85-9853-c8bd0bf0c99a

**Comment by pesty** [comment]
We experienced similar issue. We had slack integration before, and it stopped sending notifications from Oct 6th. Re-authentication doesn't work, either, as shown above.

**Comment by starjet** [comment]
Having exactly the same issue  
From what I can tell, this happened after the comments feature was added ( https://vercel.com/changelog/comments-now-available-in-vercels-slack-integration )

**Comment by ricemountainer** [comment]
I'm facing same issue. As they said on [X(twitter)](https://x.com/vercel_support/status/1711317089775231179?s=20), I believe Vercel is already working on it. I'm waiting for the fix.

**Comment by swarnava** [comment]
Hi, we are investigating this currently.

**Comment by swarnava** [comment]
Hi,

We have pushed a fix for this issue. Could you try again and let us know if you are still facing issue?

**Reply by starjet** [reply]
Okay, I managed to reauthenticate properly this time  
And I can once again see my current subscriptions via Slack so I assume it works again now (though I will have to wait for our next deployment to be certain)

**Reply by dickyindra** [reply]
Great, everything is back to normal.
Thank you, @swarnava !
